### PR TITLE
AW-100 Use Contact Us form only for Support Package requests

### PR DESCRIPTION
**Pull request by spg**

### CODE REVIEW

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/gwtpwebsite/server/support/SupportModule.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.arcbees.gwtpwebsite.server.support;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/gwtpwebsite/server/support/SupportModule.java]: Missing license header on new files

**Comment by christiangoudreau** [comment]
LGTM after nits

File: src/main/java/com/arcbees/gwtpwebsite/server/support/SupportModule.java (hunk 1 above)
Comment (spg) [inline src/main/java/com/arcbees/gwtpwebsite/server/support/SupportModule.java]: @christiangoudreau will follow in next PR

**Pull request merged**